vote.py

```diff
@@ -133,13 +133,14 @@ def askyesno(self, event):
 
     def question_result(self, event=None):
         """Return results of question"""
-        res = ''
-        for k, v in self.answers.iteritems():
-            res += '{0} -> {1}, '.format(v[0], v[1])
+        def f(item):
+            return '{0} -> {1}'.format(item[0], item[1])
+
+        res = list(map(f, self.answers.values()))
```

Comment:
@Adman doesn't this just look much better? Also, you could probably do the join right here.